packages/currency/src/chains/evm/aurora-testnet.ts

```diff
@@ -0,0 +1 @@
+export const chainId = 1313161555;
```

Comment:
(Repeated comment)
Mind the file location, should ideally be moved out of `evm` until we fix the Near identification.